fontbakery-check-ttf.py

```diff
@@ -3814,6 +3903,36 @@ def check_font_em_size_is_ideally_equal_to_1000(fb, font, skip_gfonts):
       fb.ok("Font em size is equal to 1000.")
 
 
+def check_regression_v_number_increased(fb, new_font, old_font, f):
+  fb.new_check("Version number has increased since previous release?")
+  new_v_number = new_font['head'].fontRevision
+  old_v_number = old_font['head'].fontRevision
+  if new_v_number < old_v_number:
+    fb.error(("Version number %s is less than or equal to"
+              " old version %s") % (new_v_number, old_v_number))
+  else:
+    fb.ok(("Version number %s is greater than"
+           " old version %s") % (new_v_number, old_v_number))
+
+
+def check_regression_glyphs_structure(fb, new_font, old_font, f):
+  fb.new_check("Glyphs are similiar to old version")
+  bad_glyphs = []
+  new_glyphs = glyphs_surface_area(new_font)
+  old_glyphs = glyphs_surface_area(old_font)
+
+  shared_glyphs = set(new_glyphs) & set(old_glyphs)
+
+  for glyph in shared_glyphs:
+    if int(new_glyphs[glyph]) ^ int(old_glyphs[glyph]) > 8000:
```

Comment:
That's not much readable. Instead of bitwise-XOR-ing the integer values, I think that a similar, but more readable aproach would be to do something like abs(int(v1) - int(v2)) > threshold

So, this would be:
```python
if abs(int(new_glyphs[glyph]) - int(new_glyphs[glyph])) > 8000:
```